app/services/googlepass/GooglePassUtil.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.googlepass
+
+import com.google.auth.oauth2.GoogleCredentials
+import config.AppConfig
+import models.{GooglePassCard, GooglePassTextRow}
+
+import java.util.UUID
+import javax.inject.Inject
+
+class GooglePassUtil @Inject()(config: AppConfig, createGenericPrivatePass: CreateGenericPrivatePass) {
+
+  val issuerId: String = config.googleIssuerId
+  val id: String = s"hmrc-${UUID.randomUUID()}"
+  val key: String = config.googleKey
+  val expiry: Int = config.googleJWTExpiry
+
+
+  def createGooglePassWithCredentials(name: String, nino: String, googleCredentials: GoogleCredentials): String = {
+
+    val googlePassCardContent = createGooglePassCardContent(name, nino)
+    val jwt = createGenericPrivatePass.createJwtWithCredentials(id, issuerId, googlePassCardContent, googleCredentials, expiry)
+    val saveUrl = "https://pay.google.com/gp/v/save/" + jwt
```

Comment:
I saw some hard coded strings in this file. Is it possible to extract them into configuration or messages file?